MAUI/Maps/tooltip.md

```diff
@@ -382,9 +382,9 @@ public class Model
 ![Maps tooltip settings](images/tooltip/tooltip-appearance-customization.png)
 
 N>
-* Refer to the `ShapeTooltipSettings` to customize the shape tooltip appearance.
-* Refer to the `BubbleTooltipSettings` to customize the bubble tooltip appearance.
-* Refer to the `MarkerTooltipSettings` to customize the marker tooltip appearance.
+* Refer to the [`ShapeTooltipSettings`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapShapeLayer.html#Syncfusion_Maui_Maps_MapShapeLayer_ShapeTooltipSettings) to customize the shape tooltip appearance.
+* Refer to the [`BubbleTooltipSettings`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapShapeLayer.html#Syncfusion_Maui_Maps_MapShapeLayer_BubbleTooltipSettings) to customize the bubble tooltip appearance.
+* Refer to the [`MarkerTooltipSettings`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapLayer.html#Syncfusion_Maui_Maps_MapLayer_MarkerTooltipSettings) to customize the marker tooltip appearance.
```

Comment:
API links not included in the below NOTE section

N> The `BindingContext` of the Tooltip will be the `MapTooltipInfo`, and it has the `DataItem` property. This property holds the corresponding underline object for the shape, bubble tooltip, and the corresponding `MapMarker` for the marker tooltip.

Reply:
I have included.